provider/resources.go

```diff
@@ -130,6 +111,30 @@ func Provider() tfbridge.ProviderInfo {
 					},
 				},
 			},
+			"azuredevops_serviceendpoint_generic":           {Tok: makeResource("ServiceEndpointGeneric")},
+			"azuredevops_serviceendpoint_generic_git":       {Tok: makeResource("ServiceEndpointGenericGit")},
+			"azuredevops_serviceendpoint_github":            {Tok: makeResource("ServiceEndpointGitHub")},
+			"azuredevops_serviceendpoint_github_enterprise": {Tok: makeResource("ServiceEndpointGitHubEnterprise")},
+			"azuredevops_serviceendpoint_gitlab": {
```

Comment:
Why was this one manually added and not the other new resources? I guess it has to do with the docs?

There's some inconsistencies now though. The GitLab one is called `ServiceEndpointGitLab` whereas the other new resources like the Snyk one are called `ServiceendpointNAME`. Seems like the provider is already using mixed casing for similar resources right now, but should we decide on a single one going forward?

Reply:
It had to be manually added because the docs file name doesn't match the resource name: 

> azuredevops_serviceendpoint_gitlab: could not find docs for resource 'azuredevops_serviceendpoint_gitlab'. Override the Docs property in the resource mapping.

Reply:
Regarding the naming, you are absolutely right but unfortunately I missed the wrong "Serviceendpoint" ones. I guess since it's not released yet I could cancel the release pipeline.